src/dex-ui/routes/router.tsx

```diff
@@ -119,6 +119,14 @@ export const router = createBrowserRouter(
           <Route path={Paths.DAOs.Members} element={<MembersList />} />
           <Route path={Paths.DAOs.Settings} element={<DAOSettings />} />
         </Route>
+        <Route
```

Comment:
1) I believe we want the DAO Details Proposal to show in the list of proposal types.
![Screen Shot 2023-07-25 at 3 01 45 PM](https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/54907098/9f24c0e3-9711-49ec-9c8a-8fd33f4459b1)

2) The "Change" button should navigate the user to the second step in the new proposal wizard with the type preselected.

Reply:
_A note about point 2 above:_ I left a note in Figma for Tunde to determine if we should hide the "Type" step in the Create Proposal wizard when a user navigates to the wizard using a shortcut button. For now, we should follow the same pattern we're using for other proposal wizard shortcut buttons. If needed, we can hide the "Type" step in another task.

Reply:
@damiano-melcarne-lab49 I have addressed all feedbacks just for above one I am planning to do it in separate PR, if that is fine.?

cc: @amandeeprehal-lab49